reactor-core/src/main/java/reactor/core/publisher/Tracer.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) 2011-2018 Pivotal Software Inc, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package reactor.core.publisher;
+
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+import sun.misc.JavaLangAccess;
+import sun.misc.SharedSecrets;
+
+/**
+ * Utility class for the call-site extracting.
+ *
+ * @author Simon Baslé
+ * @author Sergei Egorov
+ */
+final class Tracer {
```

Comment:
Maybe not Tracer to keep that naming space for tracing (aka zipkin) features

Reply:
I removed the class, not needed anymore. The content moved to `Traces` where it belongs :)